### DISCUSSION TITLE
REST API Webservice - TOKEN

### DISCUSSION THREAD

**Discussion by root2k6** [YetiForceCompany/YetiForceCRM#17009] [How to]
How can I generate TOKEN for Webservice Standard type. 
I have username, password and key, but I dont know, how can I generate token for this service.

**Comment by mariuszkrzaczkowski** [comment]
https://doc.yetiforce.com/api/#/Users/1ac849ff510b6b65243b46ffb9e48a9b

**Comment by root2k6** [comment]
Thank you for answer, but when I do like in link, I have an error like this:
{"status":0,"error":{"message":"Invalid data access","code":401,"file":"api\/webservice\/WebserviceStandard\/Users\/Login.php","line":383,"backtrace":"#0 api\/webservice\/WebserviceStandard\/Users\/Login.php:383 >> Api\\Core\\Exception->__construct('Invalid data access',401)\n#1 api\/webservice\/WebserviceStandard\/Users\/Login.php:242 >> Api\\WebserviceStandard\\Users\\Login->checkAccess()\n#2 api\/webservice\/Controller.php:123 >> Api\\WebserviceStandard\\Users\\Login->post()\n#3 webservice.php:28 >> Api\\Controller->process()\n#4 public_html\/webservice.php:13 >> require('webservice.php')"}}

**Reply by mariuszkrzaczkowski** [reply]
look at this
https://github.com/YetiForceCompany/YetiForceScripts/blob/master/YetiREST/index.php#L17

**Comment by root2k6** [comment]
Fine. Its work. Thanks.
